carma_wm/src/WMListenerWorker.h

```diff
@@ -77,10 +77,14 @@ class WMListenerWorker
    */
   void setRouteCallback(std::function<void()> callback);
 
+  void setConfigSpeedLimitCallback(double config_lim);
```

Comment:
add brief description, and this doesn't look like it's setting a callback, so just setConfigSpeedLimit is fine